### ISSUE TITLE
Libraries with mixed portrait/landscape boxarts looks messy

### ISSUE THREAD

**Issue by AndersHP84** [mmatyas/pegasus-frontend#124]
In particular with SNES games; European games are in landscape and Japanese games are in portrait. 

It would be nice if Pegasus recognized aspect ratio and fixed the height of the portrait boxes. 
Another solution could be to rotate the portrait boxarts to make them landscaped.

**Comment by mmatyas** [comment]
Sorry for the late reply, yes, this is an interesting issue: the original idea was to display portrait boxarts in 5 columns and landscapes in 4; if there are mixed kind of images, that's the case where I'm not sure how to avoid looking ugly.

To make them fit into the same row, one of the two kinds would have to be scaled down, making them uneven or introduce wasted spaces inside the grid. I also don't want to rotate landscapes, because as a SNES player I want to see the boxes horizontally, yet not too small, which has lead me to the current solution.

Hmm...

**Comment by AndersHP84** [comment]
I have to say that after testing some more, maybe your current solution isn't bad at all. 
It would be looking weird if portrait boxes got Black bars in the side. 

Maybe you could just crop out the bottom of portrait boxes instead of having them appearing 'behind' the next row? And when highlighting, they could appear in full portrait, the way they currently do?

**Comment by mmatyas** [comment]
Yes, I think that could be done. The interesting part would be restoring the highlighted image, but that should work too. I'll experiment with it.

**Comment by AndersHP84** [comment]
OK for me. It turned out that this is so rare that I forgot that I actually opened this issue.
